src/clients/java/src/test/java/com/tigerbeetle/AsyncRequestTest.java

```diff
@@ -20,51 +20,50 @@ public class AsyncRequestTest {
 
     @Test
     public void testCreateAccountsRequestConstructor() {
+        try (var client = getDummyClient()) {
```

Comment:
Why don't we use a single static client here, like we did for the Integration tests?

Reply:
We _don't_ need single client here, as echoClient is cheap and we don't need to amortise. And with many clients, we exercise open/close code path more, which seems like a good thing. With a single client we probably wouldn't have noticed the leak here at all.